src/arch/riscv/vm.c

```diff
@@ -46,6 +46,9 @@ void vcpu_arch_reset(struct vcpu* vcpu, vaddr_t entry)
 
     if (CPU_HAS_EXTENSION(CPU_EXT_SSTC)) {
         csrs_stimecmp_write(~0U);
+        csrs_henvcfg_set(HENVCFG_STCE);
+    } else {
+        csrs_henvcfg_clear(HENVCFG_STCE);
```

Comment:
Is this clear necessary?

"These STCE bits are WARL and are hard-wired to 0 when this extension is not implemented."

Reply:
This is not really relevant for this PR. But I believe this is already fixed in the main branch.